README.md

```diff
@@ -99,19 +104,18 @@ Follow along as I use mixins to make our grid system more semantic.
 
 ```html
 <div class="container">
-  <div class="row article">
```

Comment:
Don't remove this class. It's a talking point for the HTML5 stuff below. This is here to emphasize semantic markup.